mkdocs/docs/getting-started/identity/onchain-identity.md

```diff
@@ -0,0 +1,24 @@
+# OnChain Identity
+
+OnChain Identity is an identity that is controlled by a Smart Contract. It is a special case of Ethereum-controlled Identity.
+
+OnChain Identity manages own identity trees on chain. It can issue credentials (by adding them into it's claims tree), revoke them and perform state transitions.
```

Comment:
```suggestion
The OnChain Identity manages its own identity trees on chain. It can issue credentials (by adding them to its claims tree), revoke them and perform state transitions.
```